### PR TITLE
[Enhancement] set initial layer config when set layer type

### PR DESCRIPTION
**Pull request by jwasilgeo**
Signed-off-by: Shan He <heshan0131@gmail.com>

[Bug] fix setting layer type override layer name

Signed-off-by: Shan He <heshan0131@gmail.com>
Signed-off-by: Jacob Wasilkowski <jwasilkowski@foursquare.com>

### CODE REVIEW

**Review by heshan0131 — APPROVED** [review]

**Comment by heshan0131** [comment]
please make sure to remove (UN-1863) from the final squashed commit message

**Review by igorDykhta — APPROVED** [review]

File: src/layers/src/h3-hexagon-layer/h3-hexagon-layer.ts [hunk 1]
```diff
@@ -172,6 +172,9 @@ export default class HexagonIdLayer extends Layer {
   }
 
   setInitialLayerConfig(dataset) {
+    if (!dataset.allData?.length) {
```
Comment (igorDykhta) [inline src/layers/src/h3-hexagon-layer/h3-hexagon-layer.ts:175]: dataset.dataContainer.runRows()

File: src/layers/src/point-layer/point-layer.ts [hunk 2]
```diff
@@ -207,6 +207,9 @@ export default class PointLayer extends Layer {
   }
 
   setInitialLayerConfig(dataset) {
+    if (!dataset.allData?.length) {
```
Comment (igorDykhta) [inline src/layers/src/point-layer/point-layer.ts:210]: dataset.dataContainer.runRows()

**Review by igorDykhta — CHANGES_REQUESTED** [review]

**Comment by heshan0131** [comment]
Same here...remove UN-1863 from squashed message

**Comment by jwasilgeo** [comment]
Good catch, I cleaned that up @heshan0131, thanks.

**Review by igorDykhta — APPROVED** [review]

File: src/layers/src/point-layer/point-layer.ts (hunk 2 above)
Comment (jwasilgeo) [reply src/layers/src/point-layer/point-layer.ts]: ce42561

File: src/layers/src/h3-hexagon-layer/h3-hexagon-layer.ts (hunk 1 above)
Comment (jwasilgeo) [reply src/layers/src/h3-hexagon-layer/h3-hexagon-layer.ts]: ce42561

**Pull request merged**